### ISSUE TITLE
Grieving player head blocks by placing anvils.

### ISSUE THREAD

**Issue by kuohsuanlo** [GriefPrevention/GriefPrevention#248]
I would like to report a possible way of grieving.
A random player could place an anvil at the same location of the "HEAD" block in one's claim. By doing so, the anvil is placed and quickly rejected by the GP plugin claim system. However, the head block seems to lose its meta-data and become a skeleton skull.

And it's not a packet glitch but actually rewriting the block data.
 
https://i.imgur.com/zOlAiKg.png
https://i.imgur.com/nQwBz2P.png
  

It might be spigot's bug since canceling block placed events shouldn't have differences between anvil and other types of bug. Other block like water or lava do make the same illusion of "corrupting the head meta-data", but only on client side and won't rewrite server side's meta-data.

But I've implemented a tweaks that solves the problem from plugin's side though.
Hope you could consider implement this in plugin's POV for now. It's proved working on my server. You could test it on yours too.

```
@EventHandler
public void onPlayerInteract(PlayerInteractEvent event){
    Player player = event.getPlayer();
    Material inHandItem = player.getInventory().getItemInMainHand().getType();
    if(inHandItem != Material.ANVIL) return;
    Block clickedBlock = event.getClickedBlock();
    // Pseudo
    //
    if( Check whether the clickedBlock's claim contains player's name in its build list. ){
    }
    else{
        event.setCancelled(true);
    }
    return;
}
```

**Comment by RoboMWM** [comment]
Turns out it is indeed a spigot issue, but said issue is private. Guess this'll have to be implemented in GP then.

**Comment by RoboMWM** [comment]
I'm going to assume this is still an issue with 1.13? Let me know https://github.com/TechFortress/GriefPrevention/releases/tag/16.9.alpha

**Comment by kaanakduman** [comment]
The bug appears to be patched on Spigot 1.13.1 and GriefPrevention 16.9 for me.

**Comment by RoboMWM** [comment]
Thanks! If this is still an issue feel free to comment here and I'll reopen.
